ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/Analyzer.java

```diff
@@ -668,8 +671,11 @@ private void throwOnUdafs(final Expression expression) {
         public Void visitFunctionCall(final FunctionCall functionCall, final Void context) {
           final FunctionName functionName = functionCall.getName();
           if (metaStore.isAggregate(functionName)) {
-            throw new KsqlException("Use of aggregate function "
-                + functionName.text() + " requires a GROUP BY clause.");
+            analysis.addAggregateFunction(functionCall);
```

Comment:
we should probably change the method name, right?

Reply:
yes good catch!